src/main/java/com/minecolonies/entity/pathfinding/AbstractPathJob.java

```diff
@@ -421,9 +422,17 @@ private boolean doDebugSleep()
         return false;
     }
 
+    private static BlockPos BLOCKPOS_IDENTITY = new BlockPos(0, 0, 0);
+    private static BlockPos BLOCKPOS_UP = new BlockPos(0, 1, 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "BLOCKPOS_UP" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)